### PR TITLE
Add New Benchmarks

### PR DESCRIPTION
**Pull request by morgangibbs87**
- [ ] g4dn.xlarge
- [ ] L4 (g6.xlarge like Bonsai)
- [ ] c7i.48xlarge to `nightly`?

### CODE REVIEW

File: .github/workflows/bench_pr.yml [hunk 1]
```diff
@@ -30,16 +30,23 @@ jobs:
...
           - os: Linux
             feature: cuda
-            device: nvidia_rtx_3090_ti
-          - os: macOS
-            feature: metal
-            device: apple_m2_pro
+            device: g4dn.xlarge
+          - os: Linux
+            feature: cuda
+            device: nvidia_rtx_l4
```
Comment (flaub) [inline .github/workflows/bench_pr.yml:46]: this should be named the AWS instance type

**Comment by flaub** [comment]
I don't think we need `c7i.48xlarge`. Also we should rename L4 to the AWS instance type = `g6.xlarge`

**Review by flaub — CHANGES_REQUESTED** [review]
Holding off for now

**Comment by morgangibbs87** [comment]
Closing this PR to split into separate tests.

**Pull request closed without merging**